src/gentropy/dataset/study_locus.py

```diff
@@ -920,6 +922,76 @@ def qc_redundant_top_hits_from_PICS(self: StudyLocus) -> StudyLocus:
             _schema=StudyLocus.get_schema(),
         )
 
+    def qc_explained_by_SuSiE(self: StudyLocus) -> StudyLocus:
+        """Flag associations that are explained by SuSiE associations.
+
+        Credible sets overlapping in the same region as a SuSiE credible set are flagged as explained by SuSiE.
+
+        Returns:
+            StudyLocus: Updated study locus with SuSiE explained flags.
+        """
+        # unique study-regions covered by SuSie credible sets
+        susie_study_regions = (
+            self.filter(f.col("finemappingMethod") == "SuSiE-inf")
+            .df.select(
+                "studyId",
+                "chromosome",
+                "locusStart",
+                "locusEnd",
+                f.lit(True).alias("inSuSiE"),
+            )
+            .distinct()
+        )
+
+        # non SuSiE credible sets (studyLocusId) overlapping in any variant with SuSiE locus
+        redundant_study_locus = (
+            self.filter(~(f.col("finemappingMethod") == "SuSiE-inf"))
```

Comment:
Why not `f.col("finemappingMethod") != "SuSiE-inf"`?

Reply:
```suggestion
            self.filter(f.col("finemappingMethod") != "SuSiE-inf")
```